### ISSUE TITLE
Pusher is not working on trading page

### ISSUE THREAD

**Issue by arshad-bilawat** [openware/peatio#538]
Hi,

The pusher is not working on trading page. When I add a new order, I need to refresh the page manually.  http://localhost:3000/markets/btcusd

However, If I work in the funds page, Add USD etc, The pusher is working properly.
http://localhost:3000/funds

I checked the log in my pusher app, When ever I try to create any transaction in trade page, I get below log in pusher debug console.

DISCONNECTION
234044.370865

Channels: market-global, market-btcusd-global, private-PEAIM6XA7EHTIO, Lifetime: 223.468643334s

**Comment by yivo** [comment]
Hello @arshad-bilawat 

Have you updated configs with your own pusher access tokens?

**Comment by arshad-bilawat** [comment]
@yivo Yes, I have updated the config with my pusher settings and is working fine is funds page. It is not working in trading page and when I check the debug logs in my pusher account, it shows disconnection immediately after I initiate any transaction on the page.

**Comment by arshad-bilawat** [comment]
Hello @yivo, any update or direction around this issue please..

**Comment by yivo** [comment]
@arshad-bilawat Hi. We have actually managed to get it work by putting correct access credentials and configs. We tested it in Chrome. Everything works fine. What OS and browser do you use for tests?

**Comment by arshad-bilawat** [comment]
I am using macOS 10.13.3 and chrome. It works fine in funds page, so I am not sure if it credential issue.

**Comment by yivo** [comment]
Hi @arshad-bilawat,

We also use latest OS X & Chrome for testing but we don't expect such troubles like yours.

Sorry, but we need more details for this issue. I assume it is configuration problem (not only Pusher keys but also all yml files) so I am closing this issue. Please, reopen if you have more details. Thank for your feedback!

**Comment by Sambusik** [comment]
Hi @yivo 

Having the same issue.

I've updated `application.yml` with my own pusher keys.

It was giving issues in the console when using `PUSHER_HOST: api.pusherapp.com`, so I updated it to: `PUSHER_HOST: ws.pusherapp.com` (note ws instead of api), this removed the errors in the console.

Next I added `PUSHER_CLUSTER: 'us2'` and made sure it was being used as well in `application_controller.rb` & `config/initializers/pusher.rb`

However, For Funds (when receiving an incoming transaction for example) & in Trade page (making an order) the page does not update, unless I do a manual Refresh to the pages.

Mac OSX / Chrome (latest)
